commit d65c6a7ac821f5672d2b7cf54bea1f3c3f003b4e (HEAD -> master)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-08-12T11:03:33+01:00 (8 weeks ago)
    
    TOOLS-2297 update deps/eng for copyright checks and motd work
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/deps/eng b/deps/eng
index ca4a08e..9058562 160000
--- a/deps/eng
+++ b/deps/eng
@@ -1 +1 @@
-Subproject commit ca4a08e8b7bba7a86d897060338fba27ee4e872b
+Subproject commit 9058562f2f59892795e470ce25443ff636c90478
